### PR TITLE
Update dependencies and packages in nginx-ingress v1.1.1 images

### PR DESCRIPTION
**Pull request by bhatpmk**
Updates dependencies and packages in nginx-ingress v1.1.1 images

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,9 +16,10 @@ require (
 	github.com/mitchellh/go-ps v1.0.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/mitchellh/mapstructure v1.4.3
+	github.com/moul/pb v0.0.0-20220425114252-bca18df4138c
 	github.com/ncabatoff/process-exporter v0.7.10
 	github.com/onsi/ginkgo v1.16.4
-	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runc v1.1.2
```
Comment (kminder) [inline go.mod:22]: Do we have PLS approval for this version?

File: images/custom-error-pages/rootfs/go.mod [hunk 2]
```diff
@@ -10,11 +10,12 @@ require github.com/prometheus/client_golang v1.11.1
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.1.1 // indirect
-	github.com/golang/protobuf v1.4.3 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
```
Comment (kminder) [inline images/custom-error-pages/rootfs/go.mod:13]: PLS approval?

**Review by kminder — APPROVED** [review]
Approved.

File: go.mod (hunk 1 above)
Comment (bhatpmk) [reply go.mod:19]: There is no change in the total transitive dependency list

File: images/custom-error-pages/rootfs/go.mod (hunk 2 above)
Comment (bhatpmk) [reply images/custom-error-pages/rootfs/go.mod:10]: There is no change in the total transitive dependency list

**Pull request merged**